### DISCUSSION TITLE
Error en Lab04.01 Task 4

### DISCUSSION THREAD

**Discussion by ddavidSM** [rramosp/ai4eng.v1#105] [04 - GESTIÓN DE DATOS]
Saludos, estoy teniendo un problema al entregar la Task 4 del Lab04.01. En dicha tarea se pide hacer un dataset con el que se da de entrada, para este caso yo no estoy cargando datos sino que lo hago en el mismo notebook en python, sin embargo, cuando entrego la tarea para su calificación me sale el error de la captura adjunta.

![image](https://user-images.githubusercontent.com/95448251/149813911-ae5f5954-c74e-4658-86a0-f8122c06ba77.png)

Quedo atento a su respuesta, gracias.

**Comment by Felipe-RA** [comment]
Saludos ddavidSM

El corrector objetivo revisa que la variable objetivo (en este caso **r04** con el dataframe de pandas creado a partir del csv que le das) cumpla con las características.

No importa si este csv lo creas usando Python o lo cargas directamente como archivo, de ambas formas la siguiente celda siempre lo traduce a un `pandas.dataframe`
```
r04 = pd.read_csv("place_onehot.csv")`
r04
```


El error que muestras ahí parece ser un problema del grader de la plataforma, acabo de comprobar que ya está funcionando correctamente.

Sin embargo, si sigues teniendo problemas con este TASK vuelve a decirmelo y lo miramos  más a fondo.

Hasta pronto!

**Reply by ddavidSM** [reply]
Saludos Felipe-RA.

El error sigue persistiendo, incluso intente guardando el dataframe y hacer la lectura del mismo en esa línea de código que mencionas, sin embargo, incluso así la task saca el mismo error.

**Reply by Felipe-RA** [reply]
Saludos ddavidSM

Comparteme tu notebook a este correo, le doy una mirada profunda

felipe.rodrigueza@udea.edu.co

**Reply by rramosp** [reply]
hola david, creo que has debido de modificar algo en la celda de login. Fíjate que el error te dice que está buscado el grader del laboratorio L04.01d, con una "d" al final y ese laboratorio no existe.

Asegúrate que la celda de login es como la original de ese notebook

![image](https://user-images.githubusercontent.com/13835425/150131618-e75c1977-595f-40c0-87a6-9667e61fd304.png)

**Reply by ddavidSM** [reply]
En efecto ese era el problema, ya quedo solucionado, muchas gracias.
